prometheus/src/main/java/org/opensearch/sql/prometheus/functions/resolver/QueryExemplarsTableFunctionResolver.java

```diff
@@ -0,0 +1,48 @@
+package org.opensearch.sql.prometheus.functions.resolver;
```

Comment:
Please add license header

Reply:
we need to add license header check in gradle. will pick that up later.